### DISCUSSION TITLE
Estimated end time just under "Total time"

### DISCUSSION THREAD

**Discussion by Triskae** [OrcaSlicer/OrcaSlicer#6148] [Ideas]
Wouldn't it be great to have just below "Total time", "Estimated end time", which would give us the estimated date and time of the end of printing? A bit like on GPS devices?
I know you only need to add the "Total time" to the current date and time, but it's really for the sake of convenience.

**Comment by lin-ycv** [comment]
unless you print instantly after slicing, there's really no point. you could be printing a 3 hour print 2 hours from now, the slicer wouldn't know when you plan to start a print.
while most new printers have wifi capabilities, there's still a lot pre-existing printers that don't. and if your printer have wifi capabilities, it'll most likely be able to tell you the estimated end time.
also the estimated total time is not 100% accurate to start with, just ball park guessing the end time using total time is probably enough for most people.
